test-apps/ninezone-test-app/src/pages/Home/Navigation.tsx

```diff
@@ -3,7 +3,7 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 import "./Navigation.scss";
-import classnames from "classnames";
+import * as classnames from "classnames";
```

Comment:
Did you have to change all of these imports?

Reply:
It was either this or changing our tsconfig. Apparently the default export was a bug according to them. I didn't want to risk affecting other imports by changing tsconfig options.

Reply:
gotcha, that's annoying...